### PR TITLE
clang-format test

### PR DESCRIPTION
**Pull request by bridiver**
## Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Windows
  - [ ] macOS
  - [ ] Linux
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests && npm run test-security`) on
  - [ ] Windows
  - [ ] macOS
  - [ ] Linux
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [ ] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Defines the Chromium style for automatic reformatting.
```
Comment (bridiver) [inline .clang-format]: I'll make a note here that this is copied from src/.clang-format unless anyone knows if there's a way to also use the parent config

File: app/resources/js/tag_manager_polyfill.js [hunk 2]
```diff
@@ -1 +1,6 @@
-(function() { var noopfn = function() { ; }; window.ga = window.ga || noopfn; })();
+(function() {
```
Comment (bridiver) [inline app/resources/js/tag_manager_polyfill.js]: this file should be ignored

File: app/resources/js/tag_services_polyfill.js [hunk 3]
```diff
@@ -1 +1,127 @@
...
+(function() {
```
Comment (bridiver) [inline app/resources/js/tag_services_polyfill.js]: this file should be ignored

File: .clang-format (hunk 1 above)
Comment (bridiver) [reply .clang-format]: will probably just end up scripting this to either add the options on the command line or copy the file and append

File: .clang-format [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+IPC_PROTOBUF_MESSAGE_TRAITS_END|\
+IPC_STRUCT_END|\
+IPC_STRUCT_TRAITS_END|\
+POLPARAMS_END|\
+PPAPI_END_MESSAGE_MAP$"
+
+# TODO: Remove this once clang-format r357700 is rolled in.
+JavaImportGroups: ['android', 'androidx', 'com', 'dalvik', 'junit', 'org', 'com.google.android.apps.chrome', 'org.chromium', 'java', 'javax']
+
+BinPackArguments: false
```
Comment (bridiver) [inline .clang-format]: these are the options we decided to add for params

**Comment by bsclifton** [comment]
Closing as stale

**Pull request closed without merging**